share/completions/black.fish

```diff
@@ -0,0 +1,23 @@
+complete -c black -s c -l code -d 'Code to format'
+complete -c black -s l -l line-length -d 'Characters per line'
+complete -c black -s t -l target-version -d 'Python versions supported by output'
+complete -c black -l pyi -d 'Format all input files like typing stubs regardless of file extension'
+complete -c black -s S -l skip-string-normalization -d 'Don\'t normalize string quotes or prefixes'
+complete -c black -s C -l skip-magic-trailing-comma -d 'Don\'t use trailing commas as a reason to split lines'
```

Comment:
We can use double quotes to avoid the need to escape the single quote in `Don't`. Of course that's purely a matter of taste.